src/symbols.js

```diff
@@ -879,18 +868,9 @@ for (let i = 0; i < 10; i++) {
 // but they are not actually in the font, nor are they supported by the
 // Unicode accent mechanism, so they fall back to Times font and look ugly.
 // TODO(edemaine): Fix this.
-export const extraLatin = "ÇÐÞçþ";
```

Comment:
Having unicode codes is safer since it avoids accidentally using the wrong symbol.  Is that what prompted these changes?

Reply:
@kevinbarabash I didn't think of that, but that would be another upside. This is for ASCII-safety and consistency with other symbols.